oaknational/Oak-Web-Application
# Make docx cover page keystage aware

## Description
Make the cover page of the curriculum downloads (docx) keystage aware

## Issue(s)
Fixes `CUR-984`

## How to test

1. Go to https://deploy-preview-2889--oak-web-application.netlify.thenational.academy/teachers/curriculum/
2. Test downloads for both cycle 1 and cycle 2 enabled 
3. Single key stage subjects should be now correct
